core/bentley/src/Logger.ts

```diff
@@ -6,20 +6,14 @@
  * @module Logging
  */
 
-import { BeEvent } from "./BeEvent";
-import { BentleyError, getErrorMessage, getErrorMetadata, getErrorStack, GetMetaDataFunction, IModelStatus } from "./BentleyError";
+import { BentleyError, ExceptionMetaData, IModelStatus } from "./BentleyError";
 import { BentleyLoggerCategory } from "./BentleyLoggerCategory";
 import { IDisposable } from "./Disposable";
 
 /** Defines the *signature* for a log function.
  * @public
  */
-export type LogFunction = (category: string, message: string, metaData?: GetMetaDataFunction) => void;
-
-/** Defines log filter intercept.
- * @internal
- * */
-export type LogIntercept = (level: LogLevel, category: string, message: string, metaData?: GetMetaDataFunction) => boolean;
+export type LogFunction = (category: string, message: string, metaData: ExceptionMetaData) => void;
```

Comment:
Using the name `ExceptionMetaData` is kinda confusing to me here...  It's perfectly valid for something like an info log to include this - it doesn't always come from an exception.

What if we renamed it to `LogMetaData` or `LoggableMetaData` instead?

Reply:
I made it `LoggingMetaData`. Do you like that better?

Reply:
yeah sounds good